src/gui/src/gui.cpp

```diff
@@ -208,15 +238,19 @@ static void resetConversions()
       = [](const std::string& value, bool*) { return 0; };
 }
 
-Gui* Gui::singleton_ = nullptr;
-
 Gui* Gui::get()
 {
-  if (singleton_ == nullptr) {
-    singleton_ = new Gui();
+  if (!enabled()) {
+    ord::OpenRoad::openRoad()->getLogger()->warn(
+        utl::GUI,
+        150,
+        "Internal Error: gui::Gui::Get() called when enabled() is false\n{}",
+        GetSymbolizedStackTraceAsString(10, 0, true));
```

Comment:
wont this always trigger on the first get? The GUI class should exits before the main_window.

Reply:
I think the desire is that the main window should exist before the GUI class per https://github.com/The-OpenROAD-Project/OpenROAD/issues/8748